src/DynamicFormControl.vue

```diff
@@ -43,6 +48,11 @@
             formControl: Object,
             colWidth: String
         },
+        methods: {
+            valueIsEmpty(value){
+                return valueIsEmpty(value)
```

Comment:
I'm not sure if there is a way to avoid redundant syntax here. Vue wouldn't let me use the utils function directly within the html without adding it to the vue instance first.

Reply:
I think it should work if you put the import before the template. But I also think it's ok to restrict method calls to come from the component.

Reply:
The way to share code like this is with mixins :) https://vuejs.org/v2/guide/mixins.html